Documentation/building/cross-platform-testing.md

```diff
@@ -24,13 +24,15 @@ instructions assume you are building for Linux, but are easily modifiable for OS
    respect to object sizes and layout so you need to ensure you have either a
    release coreclr and release mscorlib or debug coreclr and debug mscorlib.
 3. A Linux build of CoreFX. We currently have experimental support for building
-   CoreFX on Linux via `build.sh`. The other option is to build the managed
-   parts of CoreFX on Windows, To do so run `build.cmd /p:TargetOS=Linux
-   /p:OSGroup=Linux`. It is okay to build a Debug version of CoreFX and run it
+   CoreFX on Linux via `build.sh`. 
+   The other option is:
+
+   * Build the managed parts of CoreFX on Windows. To do so run `build-managed.cmd -os=Linux -target-os=Linux`. It is okay to build a Debug version of CoreFX and run it
```

Comment:
I don't have a clear distinction either. Same with the `FilterToOsGroup` variable. 
@weshaggard do you know the difference?